en/flight_controller/pixhawk4.md

```diff
@@ -58,6 +58,18 @@ Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html).
 
 Download *Pixhawk 4* pinouts from [here](http://www.holybro.com/manual/Pixhawk4-Pinouts.pdf).
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | GPS
+USART2 | /dev/ttyS1 | TELEM1 (flow control)
+USART3 | /dev/ttyS2 | TELEM2 (flow control)
+UART4 | /dev/ttyS3 | TELEM4
+USART6 | /dev/ttyS4 | TELEM3 (flow control)
+UART7 | /dev/ttyS5 | ?
```

Comment:
Do we know what UART7 port is? Marked as "?"

Reply:
@hamishwillee 

USART6 | /dev/ttyS4  is RC SBUS
UART7 | /dev/ttyS5  is Debug Console
UART8 | /dev/ttyS6  is PX4IO